pkg/config/runtime.go

```diff
@@ -42,35 +45,102 @@ func newRuntime(v *Validated, evaluator expr.PredicateEvaluator) *runtime {
 	}
 }
 
-// ResolveFn is a function that returns a list of Combined configs for an
-// attribute bag and aspect kind set pair. It is used, at runtime, to retrieve
-// the set of configs that apply to an operation.
-type ResolveFn func(bag attribute.Bag, set KindSet) ([]*pb.Combined, error)
```

Comment:
This might still be useful down the road, though this probably doesn't matter at the moment, as it wasn't ultimately used as originally intended. The idea was to constrain Resolve and ResolveUnconditional to expose the same signature.